dataclients/kubernetes/testdata/ingressV1/tls/tls-no-secret.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
We don't read Deployment so you can also drop these yaml objects in tests.

Reply:
Also removed from existing Ingress test fixtures.